src/stores/winesStore.js

```diff
@@ -1,103 +1,116 @@
 import { defineStore } from 'pinia'
-import { ref, computed } from 'vue'
+import { computed, ref } from 'vue'
 
 export const useWinesStore = defineStore('wines', () => {
-  const wines = ref([
-    {
-      id: 0,
-      name: 'Cabernet Sauvignon',
-      type: 'Vörös',
-      style: 'Száraz',
-      price: '80-130',
-      flavor: 'Fűszeres',
-      ratings: [],
-      is_confirmed: true,
-    },
-    {
-      id: 1,
-      name: 'Chardonnay',
-      type: 'Fehér',
-      style: 'Félszáraz',
-      price: '50-80',
-      flavor: 'Gyümölcsös',
-      ratings: [],
-      is_confirmed: true,
-    },
-    {
-      id: 2,
-      name: 'Tokaji Aszú',
-      type: 'Fehér',
-      style: 'Édes',
-      price: '>130',
-      flavor: 'Egyéb',
-      ratings: [],
-      is_confirmed: true,
-    },
-    {
-      id: 3,
-      name: 'Kékfrankos Rosé',
-      type: 'Rozé',
-      style: 'Száraz',
-      price: '20-50',
-      flavor: 'Virágos',
-      ratings: [],
-      is_confirmed: true,
-    },
-    {
-      id: 4,
-      name: 'Merlot',
-      type: 'Vörös',
-      style: 'Félédes',
-      price: '50-80',
-      flavor: 'Földes',
-      ratings: [],
-      is_confirmed: true,
-    },
-  ])
-  const nextId = ref(wines.value.length)
-  function addRating(wineName, rating, comment) {
-    const wine = wines.value.find((w) => w.name === wineName)
-    if (wine) {
-      wine.ratings.push({ rating, comment })
+  const wines = ref([])
+  const confirmedWines = computed(() => wines.value.filter((w) => w.is_confirmed))
+  const pendingWines = computed(() => wines.value.filter((w) => !w.is_confirmed))
+
+  async function addRating(wineName, rating, comment) {
+    try {
+      const response = await fetch('http://localhost:3000/api/wines/rating', {
```

Comment:
you have to create a httpService.js or client.js file and that should be a wrapper for fetch, I will explain it on monday